gateway/1.5/modules/ROOT/pages/policies-included-header-removal.adoc

```diff
@@ -105,4 +104,4 @@ Because every header that matches a Header Removal policy parameter is removed f
 For example, the following parameter configuration removes any header starting with the word `Access-Control`, such as the `Access-Control-Allow-Credentials` and `Access-Control-Allow-Origin` headers from a message. The configuration also removes the `Cache-Control` header:
 
 .Inbound Header
-image::remove-inbound-header.png[]
```

Comment:
why this change?

Reply:
It's actually not a change. For some reason GitHub marked it as delete and then marked it as add in the next line.